test/integration/endpointslice/endpointsliceterminating_test.go

```diff
@@ -125,10 +126,11 @@ func TestEndpointSliceTerminating(t *testing.T) {
 				1*time.Second)
 
 			// Start informer and controllers
+			_, ctx := ktesting.NewTestContext(t)
 			stopCh := make(chan struct{})
 			defer close(stopCh)
 			informers.Start(stopCh)
-			go epsController.Run(1, stopCh)
+			go epsController.Run(ctx, 1)
```

Comment:
`stopCh` can be replaced with `ctx.Done` when `ctx` comes from `context.WithCancel`.